adjust.go

```diff
@@ -229,16 +268,67 @@ func (f *File) adjustFormula(sheet string, formula *xlsxF, dir adjustDirection,
 		formula.Content, err = f.adjustRef(strings.TrimPrefix(formula.Content, "="), dir, num, offset)
 		return err
 	}
-	if formula.Content != "" && !strings.ContainsAny(formula.Content, "[:]") {
+	if formula.Content != "" && !strings.ContainsAny(formula.Content, "[:]()") {
 		content, err := f.adjustFormulaRef(sheet, formula.Content, dir, num, offset)
 		if err != nil {
 			return err
 		}
 		formula.Content = content
+	} else if formula.Content != "" {
```

Comment:
Can we move this case into the `adjustFormulaRef` function? Parse the formula in one place to keep it simple.